config/initializers/zendesk_client.rb

```diff
@@ -6,4 +6,4 @@
   config.token = ENV.fetch('ZENDESK_TOKEN')
 end
 
-ZENDESK_GROUP_NAME = '3rd Line - Product Support (all teams)'
+ZENDESK_GROUP_ID = ENV.fetch('ZENDESK_NEW_TICKET_GROUP_ID')
```

Comment:
I think it's weird that the const is differently named to the env variable. I don't care which we use but it'd be nice if they were the same?